### DISCUSSION TITLE
Ability to add images to your profile

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#16866] [Feature Requests]
For example, I would like to add this image 

```md
![Now playing on Spotify](https://np.thomasbnt.dev/now-playing?)
```
Output : 
![Now playing on Spotify](https://np.thomasbnt.dev/now-playing?)
 

___ 

Can be a better profile with images, this would give profiles more possibilities to make their profile visitable and not flat.

**Comment by Spencer2614** [comment]
As you could use different methods for adding images. Like you we are using a single plugin at [Affordable electric bike 2023](https://www.bestelectricbikepoint.com/best-hybrid-electric-bike-2023/) for adding different images. But if you have custom website then use the custom code for adding image which you will find from google.

**Reply by thomasbnt** [reply]
Spam.
